components/team/MemberCard.tsx

```diff
@@ -0,0 +1,18 @@
+import { Card, CardContent } from '@mui/material'
+
+function MemberCard({ member }) {
+
+  return (
+    <>
+      <Card  sx={{ width: 'auto', height: 'auto', m: 1}} elevation={5}>
+        <CardContent id={'member-' + member.id}>
+          <h2>{member.name}</h2>
+          <p>{member.mail}</p>
+          <p>{member.role}</p>
+        </CardContent>
+      </Card>
+    </>
```

Comment:
This entire component already exists in the project, so let's find a way to not have repetitive code.